client/signup/config/steps-pure.js

```diff
@@ -166,6 +166,11 @@ export function generateSteps( {
 			delayApiRequestUntilComplete: true,
 		},
 
+		'domains-only': {
+			stepName: 'domains-only',
+			providesDependencies: [ 'siteId', 'siteSlug', 'domainItem' ],
+		},
```

Comment:
We'll need to add in the `isDomainOnly` prop here since it's required by `DomainsStep`.